py/tests/test_processing.py

```diff
@@ -208,7 +208,8 @@ def test_validate_sampling_configuration():
                     "op": "custom",
                     "name": "event.legacy_browser",
                     "value":["ie10"]
-                }
+                },
+                "id":1
```

Comment:
This is not formatted properly, I wonder if the formatter is running on this file?

Reply:
hmm, I did run format on everything.